src/cli/cache.rs

```diff
@@ -25,28 +24,18 @@ impl<K: Hash + Eq, V: Clone> NativeChronoCache<K, V> {
     NativeChronoCache { data: Arc::new(RwLock::new(TtlCache::new(CACHE_CAPACITY))) }
   }
 }
+
```

Comment:
`src/cli/cache.rs` can be moved into `src/cache.rs` because it's not a CLI specific implementation.

Reply:
moved it to `src/native_chrono_cache.rs` because there's already a file named `src/cache.rs` which contains a `Cache` type

Reply:
this is a in-memory implementation. We can use it as a default implementation for the trait. Better we move it to `cache.rs`. There is nothing "native" about it, we can use this implementation anywhere.